components/Bios/Bio/Bio.component.js

```diff
@@ -8,9 +9,11 @@ export default function Bio({ bio, shown, img, ...props }) {
         <div className={clsx(styles.BioCard)}>
           <div className={clsx(styles.imgWrapper)}>
             <div className={clsx(styles.imgOverlay)}>
-              <img
+              <Image
                 src={bio.img}
                 alt='Some Person'
+                height='300px'
```

Comment:
I dont think you are supposed to use px when giving a size to next Image. Can you update this.